justfile

```diff
@@ -76,7 +76,7 @@ check *FLAGS:
     ')"
 
     set -x
-    cargo clippy {{ check_flags }} --features "${FEATURES}"  {{ FLAGS }}
+    cargo check {{ check_flags }} --features "${FEATURES}"  {{ FLAGS }}
     set +x
 
 alias cl := clippy
```

Comment:
Nit: Can you move this alias just after the `clippy` recipe, and the alias for the `check` recipe right after the `check` recipe?